mods/tuxemon/maps/player_house_bedroom.yaml

```diff
@@ -23,7 +23,7 @@ events:
     type: "event"
   Give Money:
     actions:
-    - variable_math money,+,1000000
+    - set_money player,10000
```

Comment:
Leave it as 1000000 it is deliberately a very large amount, because we don't want to run out before a way to make money is implement.

Reply:
No problem.